.github/workflows/deploy-docs.yml

```
@@ -0,0 +1,203 @@
+
+      - name: Comment PR with preview URL
+        if: github.event_name == 'pull_request'
+        uses: actions/github-script@v7
+        with:
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: `📖 **Documentation Preview**\n\n🔗 Preview URL: ${{ steps.deploy.outputs.url }}\n\n*Updated with commit ${{ github.sha }}*`
+            })
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)